lib/views/feedbacks_slack_view.ex

```diff
@@ -1,7 +1,17 @@
 defmodule Aprb.Views.FeedbacksSlackView do
+  alias Aprb.Service.SentimentAnalysisService
+  
+  def prefix(event) do
+    emoji = event["properties"]["message"]
+      |>SentimentAnalysisService.sentiment_score
+      |>SentimentAnalysisService.sentiment_face_emoji
```

Comment:
`mix compile` sometimes suggest some changes, kind of like rubocop lite, i learned few things by just looking at those warnings.